src/bio.c

```diff
@@ -688,6 +687,31 @@ WOLFSSL_BIO* wolfSSL_BIO_next(WOLFSSL_BIO* bio)
     return bio->next;
 }
 
+/* BIO_wpending returns the number of bytes pending to be written. */
+size_t wolfSSL_BIO_wpending(const WOLFSSL_BIO *bio)
+{
+    WOLFSSL_ENTER("BIO_wpending");
+
+    if (bio == NULL)
+        return 0;
+
+    if (bio->ssl != NULL) {
+        return (long)wolfSSL_pending(bio->ssl);
```

Comment:
wolfSSL_pending() returns the number of buffered readable plaintext bytes.  Not writable.

Reply:
Okay I'll take a look at this again. Thanks